src/js/lumo.js

```diff
@@ -107,6 +107,22 @@ export function fileExists(filename: string): boolean {
   return false;
 }
 
+export function stat(filePath: string): fs.Stats {
+  return fs.statSync(filePath);
+}
+
+export function readdir(filePath: string): string[] {
+  return fs.readdirSync(filePath);
+}
+
+export function readSourcePaths(): string[] {
+  return [...sourcePaths];
+}
+
 export function addSourcePaths(srcPaths: string[]): void {
   sourcePaths.push(...srcPaths);
 }
+
+export function removeSourcePath(srcPath: string): void {
+  sourcePaths.splice(sourcePaths.indexOf(srcPath), 1);
```

Comment:
I'm thinking if we wouldn't just benefit if the `sourcePaths` variable were a Set.
This is not really important at first, we can always fix later.